feat(changelog): iam, em and s3 (#1076)
scaleway/docs-content: changelog/storage/december2022/2022-12-07-new-permission-sets.mdx

---
title: New Object Storage permission sets are available 
status: added
author:
    fullname: 'Join the #object-storage channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2022-12-06
category: storage
product: object-storage
---

> Identity and Access Management (IAM) is now available to manage users and applications access to objects and buckets. In each Scaleway project, you now have the possibility to give permissions to create, read, edit and delete object and buckets.

Reviewer: ```suggestion
Identity and Access Management (IAM) is now available to manage users and applications' access to objects and buckets. In each Scaleway Project, you can now give permissions to create, read, edit and/or delete object and buckets.
```
